resources/less/application.less

```diff
@@ -0,0 +1,43 @@
+@import "vars";
```

Comment:
The namespacing here is pretty near of clashing with hakija-application's hakija.less. hakija.less has everything prefixed with .application__. Maybe change the prefix to differ more clearly? Also the name of this file should be something which separates it from hakija's application. virkailija-application-handling.less?